docs/architecture/navigation.rst

```diff
@@ -0,0 +1,34 @@
+Navigation
+==========
+
+Saleor gives a possibility to configure storefront navigation. It can be done in dashboard ``Navigation`` tab.
+
+You can add up to 3 levels of menu items inside every menu you create. Each menu item can point to an internal page with Category, Collection or Page, or an external website by passing an extra URL.
+
+
+Managing menu items
```

Comment:
I think these should go into a separate "Guides" section of the documentation. Let's keep architecture concise and technical and build a separate "how to" section. What do you say?

Reply:
That's a good idea, I'll create ```Guides``` section.